nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/ExtensionPointImpl.java

```diff
@@ -102,7 +103,13 @@ public Extension createExtension(Element element) {
                     }
                 }
             }
-            contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            try {
+                contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            } catch (XMapException e) {
+                throw new RuntimeException(
+                        e.getMessage() + " while processing component: " + extension.getComponent().getName().getName(),
+                        e);
```

Comment:
Not to debate, the idea was just that an exception message is out of control and can be very long (even multi line) while "Failed to process component" will be a common prefix for those kind of errors.